Dockerfile

```diff
@@ -20,13 +27,30 @@ RUN ln -sf /dev/stdout /var/log/nginx/access.log && \
 # Create directories
 RUN mkdir -p /var/www/app /usr/share/nginx_auth /usr/share/nginx_crt /opt/share/webapp
 
-#copy code and install node & bower deps
+# Trying to exploit docker container layers:
+# If the package.json file changes then Docker will re-run the npm install sequence...
```

Comment:
after chatting live with miguel, we agreed that the current strategy works ok:

i) changes in either will trigger a rebuild, ii) bower does not take that long